### DISCUSSION TITLE
keine kalenderanzeige mehr...

### DISCUSSION THREAD

**Discussion by 3aau** [RaBa64/OpenDTU#23] [Q&A]
moin 
nach update auf die 24.2.24 habe ich keine kallenderanzeige mehr auf der livedatenseite.
unter system/database sind alle date da.
irgend wer eine idee dazu?

**Comment by 3aau** [comment]
und was soll der blinkende grüne punkt oben links??????????????

**Reply by MartinNK** [reply]
Denke das ist die Statusanzeige wie am Wechselrichter.

**Comment by MartinNK** [comment]
> und was soll der blinkende grüne punkt oben links??????????????



> moin nach update auf die 24.2.24 habe ich keine kallenderanzeige mehr auf der livedatenseite. unter system/database sind alle date da. irgend wer eine idee dazu?

die Anzeige der akt. Tages (24 Stundengrafik) dauert immer 1-2 Stunden bis der erste Balken angezeigt wird nach dem Restore und update, reboot.

**Reply by 3aau** [reply]
auch nach 36 std noch nix.

**Comment by 3aau** [comment]
auch nach 2 tagen noch kein jahreskallender zu sehen.

**Comment by 3aau** [comment]
hat wer ne ahnung was diese meldung bedeutet?
DTU-Einstellungen
Die Frequenz muss zwischen 860250000 und 923500000 kHz liegen und ein Vielfaches von 250kHz betragen!

**Reply by MartinNK** [reply]
Hallo ...
wie sieht dein System aus von der Hardware?
Schau mal unter Info / System ob der NRF24Plus erkannt wird ... was tauchen in der Console für Meldungen auf?
Der Frequenzbereich wird genutzt vom NRF für die Kommunikation mit dem Hoymiles.
Überprüfe mal deine Verbindungen und die Verkabelung. Mach einen Reset auf Werkseinstellungen und konfigure nochmal neu.
Gruß
Martin

**Comment by kinichahau2023** [comment]
Moin,
hat denn nun mal jemand das database-restore getestet und kann die Funktionalität bestätigen?
Ich traue mich wegen des Warnhinweises von Ralf noch nicht recht an das Update ran... will meine bestehende database nicht zerstören...

**Reply by hoschbaer** [reply]
Bei mir hat es problemlos funktioniert.

**Reply by kinichahau2023** [reply]
Bestens! Hat alles geklappt mit Update und database.bin-restore! Vielen Dank.

Sehr zu empfehlen ! Mir gefällt es sehr gut.
Dank an Ralf. Bitte dranbleiben.

**Comment by 3aau** [comment]
moin,
mittlerweile geht das speichern der database.bin und das rückladen.
in der ersten version wurde wohl nicht die bin-version gespeichert sondern die json.
beim rückladen gab das dann kleinholz.

**Reply by stefankolling** [reply]
Ich habe die database.bin gespeichert. Wenn ich die Datenbank allerdings wiederherstellen will, wird im Auswahl-Dialog ein JSON-File erwartet. Ich habe dann trotzdem die database.bin ausgewählt und erhalte die Meldung "JSON file is formatted incorrectly". 
Wie lädt man denn dann die bin-version wieder rein?

**Comment by MartinNK** [comment]
funktioniert bei mir auch einwandfrei ... sogar 2 Sicherungen mit verschiedenen Zeiträumen per binary copy aneinandergängen und zurücksichern mit Restore funktioniert einwandfrei. Von meinem/meiner Galaxy gesendet
-------- Ursprüngliche Nachricht --------Von: kinichahau2023 ***@***.***> Datum: 02.04.24  13:57  (GMT+01:00) An: RaBa64/OpenDTU-Database ***@***.***> Cc: MartinNK ***@***.***>, Comment ***@***.***> Betreff: Re: [RaBa64/OpenDTU-Database] keine kalenderanzeige mehr...
  (Discussion #23) 
Moin,
hat denn nun mal jemand das database-restore getestet und kann die Funktionalität bestätigen?
Ich traue mich wegen des Warnhinweises von Ralf noch nicht recht an das Update ran... will meine bestehende database nicht zerstören...

**Comment by kinichahau2023** [comment]
Prima ! Danke Euch!
Dann kann ich es ja angehen...   :)

Frohes Ostern nachträglich.
